dll/win32/shell32/lang/ca-ES.rc

```diff
@@ -356,23 +356,22 @@ BEGIN
 END
 
 IDD_RUN_AS DIALOGEX 0, 0, 240, 190
-STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Run As"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "Which user account do you want to use to run this program?", -1, 10, 20, 220, 20
-    CHECKBOX "Current User %s", 14000, 10, 45, 150, 10
-    LTEXT "Protect my computer and data from unauthorized program activity", -1, 25, 57, 200, 10, WS_DISABLED
-    CHECKBOX "This option can prevent computer viruses from harming your computer or personal data, but selecting it might cause the program to function improperly.", 14001, 25, 68, 200, 30, WS_DISABLED | BS_MULTILINE
-    CHECKBOX "The following user:", 14002, 10, 100, 90, 10
+    ICON IDI_SHELL_SECURITY, -1, 8, 8, 21, 20
+    LTEXT "Which user account do you want to use to run this program?", -1, 35, 14, 200, 15
+    AUTORADIOBUTTON "&Current User %s", IDC_RUNAS_THIS, 8, 35, 150, 10, WS_GROUP
+    AUTOCHECKBOX "Protect my computer and data from unauthorized program activity\n\nThis option can prevent computer viruses from harming your computer or personal data, but selecting it might cause the program to function improperly.", IDC_RUNAS_SAFER, 18, 47, 205, 50, WS_DISABLED | WS_TABSTOP | BS_MULTILINE | BS_TOP
```

Comment:
I'm not sure the first sentence that has been added is of any use, since this is explained by the next one.
Also, you modified the english translation(s) but left out all the others.

Reply:
I did not invent any new strings! I did however add the icon to the ones with English text and not to the others because I did not want to test every single dialog to make sure the text still fits.

Reply:
I was talking about the "Protect my computer and data from unauthorized program activity" sentence.

Reply:
The thing is, I don't want to spend my time fixing broken dialogs in languages I don't understand.

Reply:
"This checkbox is currently always disabled and it will probably never be enabled."
**If we knew for sure**, that it will never be enabled, then we could strip it entirely in all rcs. Let's vote for that!

Please do not reply to my comment, but just watch who upvotes and downvotes it. If the amount of thumbsup are more than the thumbs-down, then this means "strip that from all ressources".

Reply:
'you will have to' is bullshit with the current crap we have in rc files.
They are a huge mess, and nobody should be forced to hand-edit 30+ files.